src/widgets/table/lv_table.c

```diff
@@ -997,6 +997,36 @@ static lv_result_t get_pressed_cell(lv_obj_t * obj, uint32_t * row, uint32_t * c
     lv_point_t p;
     lv_indev_get_point(lv_indev_active(), &p);
 
+    /* Calculate total height of rows so we know if the click was outside them */
+    lv_area_t area;
+    lv_obj_get_coords(obj, &area);
+
+    uint8_t idx = 0;
+    int32_t total_row_height = 0;
+    int32_t total_column_width = 0;
+
+    for(idx = 0; idx < table->row_cnt; ++idx) {
+        total_row_height += *(table->row_h);
+    }
+
+    for(idx = 0; idx < table->col_cnt; ++idx) {
+        total_column_width += *(table->col_w);
+    }
```

Comment:
Did you mean to do this?

```suggestion
    for(idx = 0; idx < table->row_cnt; ++idx) {
        total_row_height += table->row_h[idx];
    }

    for(idx = 0; idx < table->col_cnt; ++idx) {
        total_column_width += table->col_w[idx];
    }
```

Reply:
Done in [c5db141](https://github.com/lvgl/lvgl/pull/7153/commits/c5db1415dfd875328df3294dd818eabb9107aaa9)